recipes/eigen/all/conanfile.py

```diff
@@ -11,6 +11,8 @@ class EigenConan(ConanFile):
     license = "MPL-2.0"
```

Comment:
```suggestion
    license = ("MPL-2.0", "LGPL-3.0-or-later")
```

Since it contains LGPL code by default or explicit with the option, it has to be added here too

Reply:
I changed that locally and I forgot to commit it.
Give me my cone of shame please haha